examples/seismic/skew_self_adjoint/example_iso.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+from devito import configuration
+from examples.seismic import RickerSource
+from examples.seismic.skew_self_adjoint import *
```

Comment:
for homogeneity w.r.t. the rest of the code base, we don't do star imports. We rather explicitly list all the modules used. I think flake8 will complain too

Reply:
removed all * imports